nrfx/hal/nrf_ccm.h

```diff
@@ -50,14 +50,14 @@ extern "C" {
  * @brief   Hardware access layer for managing the AES CCM peripheral.
  */
 
-#if defined(CCM_TASKS_KSGEN_TASKS_KSGEN_Msk) || defined(NRF51) || defined(__NRFX_DOXYGEN__)
+#if defined(CCM_TASKS_KSGEN_TASKS_KSGEN_Msk) || defined(__NRFX_DOXYGEN__)
```

Comment:
Won't this reintroduce the problem that was fixed by https://github.com/zephyrproject-rtos/hal_nordic/pull/134?

Reply:
Thanks for spotting it